masks3d.scad

```diff
@@ -32,6 +32,8 @@
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
 //   orient = Vector to rotate top towards, after spin.  See [orient](attachments.scad#subsection-orient).  Default: `UP`
+// Side Effects:
+//   Sets the default_tag to "remove".
```

Comment:
Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."